README.md

```diff
@@ -0,0 +1,164 @@
+
+* **Backup to S3**
+
+    Store your backups in Amazon S3 or any object storage system that supports the S3 protocol. The PostgreSQL Operator can backup, restore, and create new clusters from these backups.
+
+* **Multi-Namespace Support**
+
+    You can control how the Operator leverages Kubernetes Namespaces with several different deployment models:
+
+    * Deploy the Operator and all PostgreSQL clusters to the same namespace
+    * Deploy the Operator to one Namespace, and all PostgreSQL clusters to a different Namespace
+    * Deploy the Operator to one Namespace, and have your PostgreSQL clusters managed across multiple Namespaces
+
+* **Full Customizability**
+
+    The Operator not only makes it easy to get PostgreSQL up and running on Kubernetes-enabled platforms, but also allows you to further customize your deployments, including:
+
+    * Selecting different storage classes for your primary, replica, and backup storage
+    * Select your own container resources class for each PostgreSQL cluster deployment; differentiate between resources applied for primary and replica clusters
+    * Use your own container image repository, including `imagePullSecrets` and private repositories  support
+    * Customize your PostgreSQL configuration 
+    * Bring your own trusted certificate authority (CA) for use with the Operator API server
+    * Override your PostgreSQL configuration for each cluster
+
+## Deployment Requirements
+
+The Operator is validated for deployment on Kubernetes, GKE and EKS clusters. The Operator is cloud native and storage agnostic, working with a wide variety of storage classes, hostPath, and NFS.
+
+The Operator includes various components that get deployed to your Kubernetes cluster as shown in the following diagram and detailed in the Design section of the documentation for the version you are running.
+
+The Operator is developed and tested on CentOS and RHEL platforms but is known to run on other Linux variants.
+
+### Supported Platforms
+
+The Operator is tested against the following platforms:
+
+* Kubernetes 1.17+ 
+* Google Kubernetes Engine (GKE)
+* Amazon EKS 
+
+Other Kubernetes platforms may also work but have not been specifically tested.
+
+### Storage
+
+The Operator is tested with a variety of different types of Kubernetes storage and Storage Classes, as well as hostPath and NFS.
+The variety of different Storage Classes available for Kubernetes is too wide to verify the Operator functionality in each one. With that said, the Operator is designed to be storage class agnostic and has been demonstrated to work with additional Storage Classes.
+
+## Installation
+
+The following steps will quickly make the Operator up and running with cloud native PostgreSQL:
+
+1. Clone the percona-postgresql-operator repository:
+
+    ```
+    $ git clone /https://github.com/percona/percona-postgresql-operator
+    $ cd percona-postgresql-operator
+    ```
+
+2. Deploy the operator with the following command:
```

Comment:
you need to create pgo namespace, the point 2 should be` kubectl create namespace pgo `